tests/test_clients.py

```diff
@@ -89,13 +89,40 @@ def testx509NonExistingCert(self, vo):
             BaseClient(account='root', ca_cert=self.cacert, auth_type='x509', creds=creds, vo=vo)
 
     def testClientProtocolNotSupported(self, vo):
-        """ CLIENTS (BASECLIENT): try to pass an host with a not supported protocol."""
+        """ CLIENTS (BASECLIENT): try to pass a host with a not supported protocol."""
         creds = {'username': 'ddmlab', 'password': 'secret'}
         from rucio.client.baseclient import BaseClient
 
         with pytest.raises(ClientProtocolNotSupported):
+            BaseClient(rucio_host='https://localhost', auth_host='junk://localhost', account='root', auth_type='userpass', creds=creds, vo=vo)
+
+        with pytest.raises(ClientProtocolNotSupported):
+            BaseClient(rucio_host='junk://localhost', auth_host='https://localhost', account='root', auth_type='userpass', creds=creds, vo=vo)
+
+        with pytest.raises(ClientProtocolNotSupported):
+            BaseClient(rucio_host='junk://localhost', auth_host='junk://localhost', account='root', auth_type='userpass', creds=creds, vo=vo)
+
+        with pytest.raises(ClientProtocolNotSupported):
+            # rucio_host is checked before auth_host
+            BaseClient(rucio_host='junk://localhost', auth_host='localhost', account='root', auth_type='userpass', creds=creds, vo=vo)
```

Comment:
This test should be removed imo - the `auth_host` check has no dependency on the result of the `rucio_host` check, it just happens that the `auth_host` check is written after the `rucio_host` check. We shouldn't really care to test this order against regressions